lib/stack_master/commands/apply.rb

```diff
@@ -64,7 +64,8 @@ def create_stack
           failed!("Stack creation aborted")
         end
         upload_files
-        cf.create_stack(stack_options.merge(tags: proposed_stack.aws_tags))
+        on_failure = @config.stack_defaults['on_failure'] || 'ROLLBACK'
```

Comment:
I wonder if we should add this to the stack definition (which inherits `stack_defaults`) instead? That way it can be specified per stack instead of just read from the defaults.